src/leveldb/CONTRIBUTING.md

```diff
@@ -31,6 +31,6 @@ the CLA.
 
 ## Writing Code ##
 
-If your contribution contains code, please make sure that it follows 
-[the style guide](http://google-styleguide.googlecode.com/svn/trunk/cppguide.xml).
+If your contribution contains code, please make sure that it follows
+[the style guide](http://google.github.io/styleguide/cppguide.html).
```

Comment:
Nit:
```suggestion
[the style guide](https://google.github.io/styleguide/cppguide.html).
```

Reply:
This was fixed [upstream](https://github.com/google/leveldb/pull/862) and then later made redundant by a refactor. But the [Bitcoin Core fork](https://github.com/bitcoin-core/leveldb-subtree) that we need to follow hasn't pulled either in yet.